content/en/docs/intro/using_helm.md

```diff
@@ -334,7 +334,7 @@ nodeSelector:
 
 Deeply nested data structures can be difficult to express using `--set`. Chart
 designers are encouraged to consider the `--set` usage when designing the format
-of a `values.yaml` file.
+of a `values.yaml` file  (read more about [Values Files](https://helm.sh/docs/chart_template_guide/values_files/)).
```

Comment:
Can you please use relative links in the markdown. This will enable the links to work on both GitHub and helm.sh. Relative links is what we generally use.

Reply:
👍 done @mattfarina